browser/mapping_test.go

```diff
@@ -239,3 +239,17 @@ func isCustomMapping(customMappings map[string]string, typ, method string) (stri
 
 	return "", false
 }
+
+// ----------------------------------------------------------------------------
+// JavaScript API definitions.
+// ----------------------------------------------------------------------------
+
+// keyboardAPI is the interface of a keyboard input device.
+// TODO: map this to page.GetKeyboard().
```

Comment:
Can you clarify what does this TODO mean?

Reply:
See: 5528008